pkgs/by-name/wc/wcurl/package.nix

```
@@ -0,0 +1,65 @@
+{
+  stdenv,
+  lib,
+  curl,
+  fetchFromGitLab,
+  installShellFiles,
+  nix-update-script,
+  shunit2,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

(Sorry, last suggestion)